aiida_flexpart/calculations/collect_sens.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+Calculations provided by aiida_flexpart.
+Register calculations via the "aiida.calculations" entry point in setup.json.
+"""
+from pathlib import Path
+from aiida import orm, common, engine
+import yaml
+
+with open(Path.home() / 'work/aiida-flexpart/config/params.yaml', 'r') as fp:
+    params_dict = yaml.safe_load(fp)
+
+cosmo_models = ['cosmo7', 'cosmo1', 'kenda1']
+ECMWF_models = ['IFS_GL_05', 'IFS_GL_1', 'IFS_EU_02', 'IFS_EU_01']
+
+class CollectSensCalculation(engine.CalcJob):
```

Comment:
Please write the full name, I guess `CollectSensitivitesCalculation`:
```suggestion
class CollectSensitivitesCalculation(engine.CalcJob):
```